docs/OV_Runtime_UG/model_representation.md

```diff
@@ -3,18 +3,18 @@
 In OpenVINO™ Runtime a model is represented by the `ov::Model` class.
 
 The `ov::Model` object stores shared pointers to `ov::op::v0::Parameter`, `ov::op::v0::Result` and `ov::op::Sink` operations that are inputs, outputs and sinks of the graph.
-Sinks of the graph have no consumers and are not included in the results vector. All other operations hold each other via shared pointers: child operation holds its parent (hard link). If an operation has no consumers and it's not the `Result` or `Sink` operation
-(shared pointer counter is zero), then it will be destructed and won't be accessible anymore. 
+Sinks of the graph have no consumers and are not included in the results vector. All other operations hold each other via shared pointers, in which child operation holds its parent (hard link). If an operation has no consumers and it is not the `Result` or `Sink` operation
+(shared pointer counter is zero), then it will be destructed and will not be accessible anymore. 
 
 Each operation in `ov::Model` has the `std::shared_ptr<ov::Node>` type.
 
 For details on how to build a model in OpenVINO™ Runtime, see the [Build a Model in OpenVINO™ Runtime](@ref ov_ug_build_model) section.
 
 OpenVINO™ Runtime allows to use different approaches to work with model inputs/outputs:
- - `ov::Model::inputs()`/`ov::Model::outputs()` methods allow to get vector of all input/output ports.
+ - The `ov::Model::inputs()`/`ov::Model::outputs()` methods allow to get vector of all input/output ports.
  - For a model which has only one input or output you can use methods `ov::Model::input()` or `ov::Model::output()` without arguments to get input or output port respectively.
  - Methods `ov::Model::input()` and `ov::Model::output()` can be used with index of input or output from the framework model to get specific port by index.
- - You can use tensor name of input or output from the original framework model together with methods `ov::Model::input()` or `ov::Model::output()` to get specific port. It means that you don't need to have any additional mapping of names from framework to OpenVINO, as it was before, OpenVINO™ Runtime allows using of native framework tensor names.
+ - You can use tensor name of input or output from the original framework model together with methods `ov::Model::input()` or `ov::Model::output()` to get specific port. It means that you do not need to have any additional mapping of names from framework to OpenVINO, as it was before, OpenVINO™ Runtime allows using native framework tensor names.
 
 @sphinxtabset
```

Comment:
Need to adjust the formatting of this code snippet below to make it belong to the last bullet item.